lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -2660,6 +2660,23 @@ public function setLifecycleCallbacks(array $callbacks)
     {
         $this->lifecycleCallbacks = $callbacks;
     }
+    
+    /**
+     * Removes a lifecycle callback for entities of this class.
+     *
+     * @param string $callback
+     * @param string $lifecycleEvent
+     *
+     * @return void
+     */
+    public function removeLifecycleCallback($callback, $lifecycleEvent)
```

Comment:
`$lifecycleCallbacks` is publicly available. You are using this new API to do something reeeeeeally hacky, therefore additional API for it should really be avoided.